app/gimlet/rev-b.toml

```diff
@@ -0,0 +1,551 @@
+requires = {flash = 8192, ram = 2048 }
+stacksize = 1920        # Sensor data is stored on the stack
+start = true
+
+[tasks.idle]
+path = "../../task/idle"
+name = "task-idle"
+priority = 5
+requires = {flash = 128, ram = 256}
+stacksize = 256
+start = true
+
+[config]
+
+#
+# I2C1: SPD proxy bus
+#
+[[config.i2c.controllers]]
+controller = 1
+target = true
+
+#
+# SMBUS_SPD_PROXY_SP3_TO_SP_SMCLK
+# SMBUS_SPD_PROXY_SP3_TO_SP_SMDAT
+#
+[config.i2c.controllers.ports.B]
+name = "spd"
+description = "SPD proxy"
+pins = [ { pins = [ 6, 7 ], af = 4 } ]
+
+#
+# I2C2: Front/M.2 bus
+#
+[[config.i2c.controllers]]
+controller = 2
+
+#
+# SMBUS_SP_TO_M2_SMCLK_A2_V3P3
+# SMBUS_SP_TO_M2_SMDAT_A2_V3P3
+#
+[config.i2c.controllers.ports.B]
+name = "m2"
+description = "M.2 bus"
+pins = [ { pins = [ 10, 11 ], af = 4 } ]
+muxes = [ { driver = "pca9548", address = 0x73 } ]
+
+#
+# SMBUS_SP_TO_LVL_FRONT_SMDAT
+# SMBUS_SP_TO_LVL_FRONT_SMCLK
+#
+[config.i2c.controllers.ports.F]
+name = "front"
+description = "Front bus"
+pins = [ { pins = [ 0, 1 ], af = 4 } ]
+
+#
+# Shark fin muxes
+#
+[[config.i2c.controllers.ports.F.muxes]]
+driver = "pca9548"
+address = 0x70
+
+[[config.i2c.controllers.ports.F.muxes]]
+driver = "pca9548"
+address = 0x71
+
+[[config.i2c.controllers.ports.F.muxes]]
+driver = "pca9548"
+address = 0x72
+
+#
+# I2C3: Mid bus
+#
+[[config.i2c.controllers]]
+controller = 3
+
+#
+# SMBUS_SP_TO_LVL_MID_SMCLK
+# SMBUS_SP_TO_LVL_MID_SMDAT
+#
+[config.i2c.controllers.ports.H]
+name = "mid"
+description = "Mid bus"
+pins = [ { pins = [ 7, 8 ], af = 4 } ]
+
+#
+# I2C4: Rear bus
+#
+[[config.i2c.controllers]]
+controller = 4
+
+#
+# SMBUS_SP_TO_LVL_REAR_SMCLK
+# SMBUS_SP_TO_LVL_REAR_SMDAT
+#
+[config.i2c.controllers.ports.F]
+name = "rear"
+description = "Rear bus"
+pins = [ { pins = [ 14, 15 ], af = 4 } ]
+
+[[config.i2c.devices]]
+bus = "front"
+address = 0x48
+device = "tmp117"
+name = "Southwest"
+description = "Southwest temperature sensor"
+sensors = { temperature = 1 }
+removable = true
+refdes = "J194"
+
+[[config.i2c.devices]]
+bus = "front"
+address = 0x49
+device = "tmp117"
+name = "South"
+description = "South temperature sensor"
+sensors = { temperature = 1 }
+removable = true
+refdes = "J195"
+
+[[config.i2c.devices]]
+bus = "front"
+address = 0x4a
+device = "tmp117"
+name = "Southeast"
+description = "Southeast temperature sensor"
+sensors = { temperature = 1 }
+removable = true
+refdes = "J196"
+
+[[config.i2c.devices]]
+bus = "front"
+address = 0x70
+device = "pca9545"
+description = "U.2 ABCD mux"
+refdes = "U336"
+
+[[config.i2c.devices]]
+bus = "front"
+address = 0x71
+device = "pca9545"
+description = "U.2 EFGH mux"
+refdes = "U339"
+
+[[config.i2c.devices]]
+bus = "front"
+address = 0x72
+device = "pca9545"
+description = "U.2 IJ/FRUID mux"
+refdes = "U337"
+
+[[config.i2c.devices]]
+bus = "m2"
+address = 0x73
+device = "pca9545"
+description = "M.2 mux"
+refdes = "U422"
+
+[[config.i2c.devices]]
+bus = "m2"
+mux = 1
+segment = 3
```

Comment:
I thought segments were 1-based indexes based on what I was specifying to humility. In this case that'd be 4.

Reply:
Good eyes! I really thought I had doubly checked that, but apparently not!

Reply:
Done